_posts/2021-01-12-github-application-running-on-faasd-behind-inlets.md

```diff
@@ -0,0 +1,508 @@
+---
+layout: post
+title: How to expose your Github application running locally on faasd to the internet ?
+description: In this guide, we are going to demonstrate how we can build our Github Application and run it locally on our faasd instance then expose it to the internet using inlets to be able to Github can send events to it
+tags: inlets-pro inletsctl go github-application faasd rasperry-pi caddy tls
+author_staff_member: developer-guy
+dark_background: true
+image: /images/2021-01-12-github-application-using-go-and-inlets-pro/faasd-issue-bot.png
+date: 2021-01-12
+---
+
+## Introduction
+
+In this guide, we are going to develop a [Github application](https://docs.github.com/en/free-pro-team@latest/developers/apps) using Go, then we deploy it as a serverless function to make use of [faasd](https://github.com/openfaas/faasd) which is a lightweight & portable faas engine.We are also going to do this demo on our local environment, so we should open our function which runs on our local environment to the Internet so Github can send events to our function.In order to do that we use inlets-pro which provides secure TCP/L4 tunnels.
+
+## Pre-requisites
+
+- arkade : You can basically use arkade to install of these following tools, so we need to install arkade first.
+arkade provides a portable marketplace for downloading your favourite devops CLIs and installing helm charts, with a single command.
+
+```bash
```

Comment:
I would like to avoid the user being put off by all these CLIs.

Here is my suggestion: only list the apps here, do not install all of them, and show their ASCII logo output.

Then install each one exactly and only when it's needed, to show the value of each app in context.

Does that make sense?

Reply:
done